wiki/Medals/de.md

```diff
@@ -1,74 +1,59 @@
 
-#### Anforderungen
+## Dedication
 
-- Spiele eine beliebige Schwierigkeitsstufe von jeder Beatmap eines [Theme Beatmap Packs](https://osu.ppy.sh/p/packlist?t=t)
-- Der Spielmodus muss der [osu!](/wiki/Game_mode/osu!) Modus sein.
-- Entpacke das `Theme Beatmap Pack` und packe alle Beatmaps in den osu! Song Ordner. Starte osu! und spiel die Maps. Das ist alles.
-  - Sollte die Beatmap als `.zip` Datei vorliegen, dann verschieb sie als `.zip` in den osu! Song Ordner und erstelle nicht einen neuen Ordner dafür.
-  - Die Beatmaps können auch einzeln heruntergeladen und gespielt werden.
-- Beatmaps vom selben Lied aber anderen Erstellern zählen nicht. Ausschließlich die aufgelistete Beatmap wird gewertet.
-- Man muss mit seinem Account im Spiel angemeldet sein. Nur auf der Webseite angemeldet zu sein, reicht nicht aus.
-- Mods können nach Belieben benutzt werden, außer Mods wie `Relax`, ansonsten wird der erspielte Rekord nicht gezählt.
+**Dedication Medaillen** (zu Dt. Hingabe) sind Medaillen, die vergeben werden, sobald bestimmte Meilensteine bei der Spielanzahl in einem einzigen Spielmodus erreicht werden.
 
-#### Hinweise
+## Mod Introduction
 
-- Nach Erhalten des Erfolges können die dafür benötigten Beatmaps wieder gelöscht werden. Der Erfolg, Ranked Score und die Genauigkeit bleiben erhalten. Falls man jedoch seinen Ranked Score oder die Genauigkeit verbessern möchte, dann sollte man die Beatmaps jedoch behalten oder neu herunterladen.
-- Falls man sich fragt, warum man einen der Theme Pack Erfolge nicht erhalten hat und eventuell das Gefühl hat, dass eine Map fehlt, dann kann man zu jeder Zeit die [Theme Pack Liste](https://osu.ppy.sh/p/packlist?t=t) überprüfen. Jede geschaffte Map hat ein dickgedrucktes **CLEARED** neben sich.
-- Das Emblem wird erst ausgehändigt, wenn alle Lieder aus dem Pack geschafft wurden.
-  - Der Rekord muss hochgeladen worden sein. Ansonsten kann es sein, dass die Beatmap ein weiteres Mal gespielt werden muss.
-- Bei alten Maps ist das Timing/Offset oftmals falsch. Mit dem `Offset Wizard` in den Einstellungen kann das Timing/Offset korrigiert werden.
+*Siehe auch: [Spielmodifikationen](/wiki/Game_modifier)*
 
-#### Trivial
+**Mod Introduction Medaillen** (zu Dt. Mod-Vorstellung) sind Medaillen, die beim Abschießen von [qualified](/wiki/Beatmap/Category#qualified), [ranked](/wiki/Beatmap/Category#ranked) oder [approved](/wiki/Beatmap/Category#approved) Beatmaps mit der entsprechenden [Spielmodifikation (Mod)](/wiki/Game_modifier) überreicht werden. Diese Medaillen sollen neue Spieler dazu ermutigen, jede Mod auszuprobieren und zu entdecken, was das Spiel zu bieten hat. Bedenke jedoch, dass wenn Spieler mit mehreren Spielmodifikationen zur gleichen Zeit spielen, der Spieler **nicht** berechtigt ist, *irgendeine* der Medaillen für die Spielmodifikationen für dieses Spiel zu gewinnen.
```

Comment:
```suggestion
**Mod Introduction Medaillen** (zu Dt. Mod-Vorstellung) sind Medaillen, die beim Abschließen von [qualified](/wiki/Beatmap/Category#qualified), [ranked](/wiki/Beatmap/Category#ranked) oder [approved](/wiki/Beatmap/Category#approved) Beatmaps mit der entsprechenden [Spielmodifikation (Mod)](/wiki/Game_modifier) überreicht werden. Diese Medaillen sollen neue Spieler dazu ermutigen, jede Mod auszuprobieren und zu entdecken, was das Spiel zu bieten hat. Bedenke jedoch, dass wenn Spieler mit mehreren Spielmodifikationen zur gleichen Zeit spielen, der Spieler **nicht** berechtigt ist, *irgendeine* der Medaillen für die Spielmodifikationen für dieses Spiel zu gewinnen.
```
When missing one letter changes the whole meaning of a word :sweat_smile:

Reply:
Adjusting to lazer *pew pew* 😝